custom/icds_reports/templates/icds_reports/icds_app/month-filter.html

```diff
@@ -3,6 +3,9 @@
         <h3 class="modal-title" id="modal-title">Select Month and Year<i style="float: right;" ng-click="$ctrl.close()" class="fa fa-close pointer"></i></h3>
     </div>
     <div class="modal-body" id="modal-body">
+          <div class="alert alert-info" ng-show="$ctrl.showMessage">
+      <strong>Info!</strong>  Supported period is after Feb 2019.
```

Comment:
where did you find this info message?
Asking because it looks a bit wrong. it actually starts from Feb(Including Feb). see here: https://dimagi-dev.atlassian.net/browse/ICDS-644. You should ask for the right info message which is not misleading.

Reply:
Also, is this ensured in the others parts of code? that it should be available starting from Feb(Including Feb) and not before.

Reply:
Yes, Feb 2019 is included in code, only message is misleading.